### ISSUE TITLE
Array of String - invalid render

### ISSUE THREAD

**Issue by bartoszgorka** [piotrbernad/jproto#1]
Try to render structure:
```elixir
JSONProto.Profile {
  var first_name: String,
  var last_name: String,
  var email: String,
  var missing_fields: Array<String>
}
```

Result of render
```swift
struct Profile: Codable {
	let firstName: String
	let lastName: String
	let email: String
	let missingFields: String

	enum CodingKeys: String, CodingKey {
		case firstName = "first_name"
		case lastName = "last_name"
		case email = "email"
		case missingFields = "missing_fields"
	}
}
```

Field missingFields should be parsed as Array.

Can you fix it?

All base types (e.g. String, Boolean, Integer...) has the same problem. Only JSONProto structure render correct.

**Comment by piotrbernad** [comment]
Fixed in 0.1.2

**Comment by bartoszgorka** [comment]
Thank you!
`let missingFields: [String]`
